Assets/MRTK/SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -393,9 +403,19 @@ public virtual void OnPostSceneQuery()
         {
             using (OnPostSceneQueryPerfMarker.Auto())
             {
-                if (IsSelectPressed)
+                if (dragAction != MixedRealityInputAction.None && InputSourceParent.SourceType == InputSourceType.Controller)
                 {
-                    CoreServices.InputSystem.RaisePointerDragged(this, MixedRealityInputAction.None, Handedness);
+                    if (IsDragPressed)
+                    {
+                        CoreServices.InputSystem.RaisePointerDragged(this, dragAction, Handedness);
+                    }
+                }
+                else
+                {
+                    if (IsSelectPressed)
```

Comment:
If i am reading this correctly, drag can be performed with _either_ IsDragPressed or IsSelectPressed, correct?

Reply:
Yes, for backwards compatibility - if the drag action is not set.